packages/wpcom.js/src/lib/site.credit-vouchers.js

```diff
@@ -1,58 +0,0 @@
-/**
- * SiteCreditVouchers methods
- * @param {string} sid - site id
- * @param {WPCOM} wpcom - wpcom instance
- * @returns {null} null
- */
-class SiteCreditVouchers {
-	constructor( sid, wpcom ) {
-		if ( ! sid ) {
-			throw new Error( '`site id` is not correctly defined' );
-		}
-
-		if ( ! ( this instanceof SiteCreditVouchers ) ) {
-			return new SiteCreditVouchers( sid, wpcom );
-		}
-
-		this.wpcom = wpcom;
-		this._sid = sid;
-		this.path = `/sites/${ this._sid }/vouchers`;
```

Comment:
Some of those endpoints might be unused too. Would be nice to clean them up at some point, although the impact on having unused backend endpoints code is definitely smaller due to the lack of compilation/build process.